spec.html

```diff
@@ -1024,6 +1024,7 @@ <h1>Mathematical Operations</h1>
       <emu-note>
         <p><emu-eqn>floor(_x_) = _x_ - (_x_ modulo 1)</emu-eqn>.</p>
       </emu-note>
+      <p>An <dfn id="interval">interval</dfn> from lower bound _a_ to upper bound _b_ is a possibly-infinite, possibly-empty set of numeric values of the same numeric type. A numeric value _x_ is included in the interval if it satisfies both its <em>lower bound condition</em> and its <em>upper bound condition</em>. Each bound will be described as either inclusive or exclusive, but not both. If an interval's lower bound is described as inclusive, its lower bound condition is _a_ &le; _x_. If an interval's lower bound is instead described as exclusive, its lower bound condition is _a_ &lt; _x_. If an interval's upper bound is described as inclusive, its upper bound condition is _x_ &le; _b_. If an interval's upper bound is instead described as exclusive, its upper bound condition is _x_ &lt; _b_. For example, the interval from 1 (inclusive) to 2 (exclusive) consists of all mathematical values between 1 and 2, including 1 and not including 2. An <dfn id="inclusive-interval">inclusive interval</dfn> is an interval with an inclusive lower bound and an inclusive upper bound. For the purpose of defining intervals, *-0*<sub>𝔽</sub> &lt; <!-- this comment here to avoid ecmarkup complaining about comparison to 0 --> *+0*<sub>𝔽</sub>, so, for example, an inclusive interval with a lower bound of *+0*<sub>𝔽</sub> includes *+0*<sub>𝔽</sub> but not *-0*<sub>𝔽</sub>. *NaN* is never included in an interval.</p>
```

Comment:
The first few sentences read overly verbose to me. How about

An interval from lower bound _a_ to upper bound _b_ is a possibly-infinite, possibly-empty set of numeric values of the same numeric type. A bound is either inclusive or exclusive, but not both. There are four kinds of intervals as follows.

- An interval from _a_ (inclusive) to _b_ (inclusive) includes all numeric values _x_ such that _a_ &le; _x_ &le; _b_
- An interval from _a_ (inclusive) to _b_ (exclusive) includes all numeric values _x_ such that _a_ &le; _x_ &lt; _b_
- An interval from _a_ (exclusive) to _b_ (inclusive) includes all numeric values _x_ such that _a_ &lt; _x_ &le; _b_
- An interval from _a_ (exclusive) to _b_ (exclusive) includes all numeric values _x_ such that _a_ &lt; _x_ &lt; _b_

An inclusive interval is [...]

Reply:
That also WFM. I might collapse the last sentence in:

---

- An interval from _a_ (inclusive) to _b_ (inclusive), also called an `<dfn>inclusive interval</dfn>` includes all numeric values _x_ such that _a_ &le; _x_ &le; _b_
- An interval from _a_ (inclusive) to _b_ (exclusive) includes all numeric values _x_ such that _a_ &le; _x_ &lt; _b_
- An interval from _a_ (exclusive) to _b_ (inclusive) includes all numeric values _x_ such that _a_ &lt; _x_ &le; _b_
- An interval from _a_ (exclusive) to _b_ (exclusive) includes all numeric values _x_ such that _a_ &lt; _x_ &lt; _b_

Reply:
Updated.